packages/editor/src/package/serlo-renderer.tsx

```diff
@@ -1,22 +1,31 @@
+import { createRenderers } from '@editor/editor-integration/create-renderers'
+import { editorRenderers } from '@editor/plugin/helpers/editor-renderer'
 import { StaticRenderer } from '@editor/static-renderer/static-renderer'
 import type { AnyEditorDocument } from '@editor/types/editor-plugins'
 import type { SupportedLanguage } from '@editor/types/language-data'
 
+import type { PluginsConfig } from './editor'
 import { editorData } from './editor-data'
 import { InstanceDataProvider } from '@/contexts/instance-context'
 import { LoggedInDataProvider } from '@/contexts/logged-in-data-context'
 
 export interface SerloRendererProps {
+  pluginsConfig: PluginsConfig
   language?: SupportedLanguage
   document?: AnyEditorDocument | AnyEditorDocument[]
 }
 
 export function SerloRenderer({
+  pluginsConfig,
   language = 'de',
   ...props
 }: SerloRendererProps) {
+  const { customPlugins, customRenderers } = pluginsConfig
   const { instanceData, loggedInData } = editorData[language]
 
+  const basicRenderers = createRenderers(customPlugins)
+  editorRenderers.init({ ...basicRenderers, ...customRenderers })
+
```

Comment:
It's not bundle size, `linkRenderer` is separate because `link` is not a plugin. But I guess then `customRenderers` is a confusing name. At first, I named it `customLeafRenderers`, but that's Slate-specific and maybe not 100% accurate. Could you maybe propose a structure that would make sense to you?